sky/jobs/utils.py

```diff
@@ -1146,9 +1179,16 @@ def get_job_table(cls) -> str:
         return cls._build(code)
 
     @classmethod
-    def cancel_jobs_by_id(cls, job_ids: Optional[List[int]]) -> str:
+    def cancel_jobs_by_id(cls,
+                          job_ids: Optional[List[int]],
+                          all_users: bool = False) -> str:
         code = textwrap.dedent(f"""\
-        msg = utils.cancel_jobs_by_id({job_ids})
+        try:
+            msg = utils.cancel_jobs_by_id({job_ids}, all_users={all_users})
+        except TypeError:
```

Comment:
We can handle this based on the `MANAGED_JOBS_VERSION` instead of the exception; or avoid bumping the version number?